gamemodes/zombiesurvival/gamemode/obj_entity_extend.lua

```diff
@@ -334,7 +334,7 @@ function meta:NearestBone(pos)
 end
 
 function meta:IsProjectile()
-	return self:GetCollisionGroup() == COLLISION_GROUP_PROJECTILE or self.AlwaysProjectile
+	return IsValid(self) and self:GetCollisionGroup() == COLLISION_GROUP_PROJECTILE or self.AlwaysProjectile
```

Comment:
IsValid is implied, this isn't needed, especially not the bloated global version.